src/main/java/core/basesyntax/FigureColor.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class FigureColor implements Figure {
+    private String color;
+
+    public FigureColor(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    @Override
+    public double getArea() {
+        return 0;
+    }
+
+    public void draw() {
+    }
+}
```

Comment:
The class name 'FigureColor' is not appropriate as it suggests a property rather than an object or entity. Additionally, the use of an abstract class to set behavior for classes is not recommended. Instead, consider using an interface to define behavior and separate classes to implement them, especially since the 'Figure' interface is expected to be implemented by classes representing figures, not colors.